requirements.dev.txt

```diff
@@ -6,3 +6,9 @@ sphinx
 sphinx_rtd_theme
 sphinxcontrib-plantuml
 autoapi
+ipdb
+-e git+https://github.com/HPENetworking/topology.git@master#egg=topology
+-e git+https://github.com/ocelotl/topology_docker_openswitch.git@master#egg=topology_docker_openswitch
+-e git+https://github.com/ocelotl/topology_docker_host.git@master#egg=topology_docker_host
```

Comment:
shouldn't we leave docker_host as a default node? (leave it on this repo? ) @carlos-jenkins